src/ERC20ForwarderClient.js

```diff
@@ -312,15 +312,17 @@ class ERC20ForwarderClient {
 
       if (!txGas) throw new Error("'txGas' parameter is mandatory");
 
-      if (!this.isSignerWithAccounts)
-        throw new Error(
-          "Provider object passed to Biconomy does not have user account information. Refer to docs or contact Biconomy team to know how to use ERC20ForwarderClient properly"
-        );
-
       await this.checkTokenSupport(token);
 
-      if (!userAddress)
-        userAddress = await this.provider.getSigner().getAddress();
+      if (!userAddress) {
+        if (!this.isSignerWithAccounts) {
+          throw new Error(
+            "Provider object passed to Biconomy does neighter have user account information nor userAddress is passed. Refer to docs or contact Biconomy team to know how to use ERC20ForwarderClient properly"
```

Comment:
typo in neighter 
Provider object passed to Biconomy neither has user account information not userAddress is passed. Refer to docs or contact Biconomy team to know how to use ERC20ForwarderClient properly.

Reply:
fixed

Reply:
fixed

Reply:
fixed